src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +14,11 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        UserService userService = new UserService();
+        User user = userService.findByEmail(email);
+        if (user == null) {
+            return false;
+        }
+        return user.getPassword() == password;
```

Comment:
I'd recommend simplifying it this way:
```suggestion
        return user != null && user.getPassword().equals(password);
```
Do not forget to use equals() method instead of == operator